install_config/install/docker_registry.adoc

```diff
@@ -1230,8 +1230,8 @@ Error: image anyproject/pullimagestream:latest not found
 
 To avoid it, you can:
 
-1. avoid tagging of internally managed images completely
-2. or re-push the built image to the desired namespace
+.  Avoid tagging of internally managed images completely, or
+.  Re-push the built image to the desired namespace
 link:#access-pushing-and-pulling-images[manually]
```

Comment:
I think this would work best as a paragraph (with a few minor tweaks to avoid repeated words):

" To prevent this, avoid the tagging of internally managed images completely, or re-push the built image to the desired namespace manually"